addons/web/static/src/legacy/scss/primary_variables.scss

```diff
@@ -167,3 +167,7 @@ $o-kanban-dashboard-dropdown-complex-gap: 5px !default;
 // Form view
 
 $o-form-view-sheet-max-width: 1140px !default;
+
+// User input typically entered via keyboard
+
+$kbd-box-shadow: 0px 1px 1px rgba($o-black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($o-white, 0.8);
```

Comment:
Should be in `bootstrap_overridden.scss` as it is a Bootstrap variable and not an Odoo one.